beets/library.py

```diff
@@ -856,7 +856,13 @@ def move_file(self, dest, operation=MoveOperation.MOVE):
         `operation` should be an instance of `util.MoveOperation`.
         """
         if not util.samefile(self.path, dest):
+            old_dest = dest
             dest = util.unique_path(dest)
+            log.debug(
```

Comment:
I think you may need a `if dest != old_dest` here. This may also need some careful testing to make sure it doesn't fire spuriously when these point to the same path but are just normalized differently… hopefully that doesn't happen, but a few test runs should reveal the answer!

Reply:
If you have any specific suggestions of test runs, and have the time to, please let me know.  All tests pass, so I'm assuming you mean something outside of those.

Reply:
About being "normalized differently," I'm just thinking about all the various ways in which different paths can refer to the same file. For example, `os.path.normpath` or `os.path.abspath` are different ways of transforming paths so they still refer to the same file. It's just worth double-checking to make sure that we're not doing that kind of thing when transforming the path (or that the original path is already normalized in the same way, which it probably is).

I suggested `if dest != old_dest` so that (normalization aside) this message only gets printed when there is an actual change.

For testing, all I mean is running this on a few files—perhaps in the same cases that presumably motivated you to add this log message—and make sure that the message shows up when it's supposed to (and doesn't appear when it isn't).